src/matchers/Concat.ts

```diff
@@ -61,11 +68,14 @@ export class Concat implements MatchingLogic {
                     if (isSpecialMember(stepName)) {
                         this.matchSteps.push({$id: stepName, veto: def});
                     } else {
-                        this.matchSteps.push({$id: stepName, alter: def});
+                        this.matchSteps.push({$id: stepName, compute: def});
                     }
                 } else {
                     // It's a normal matcher
-                    const named = new NamedMatcher(stepName, toMatchingLogic(def));
+                    const m = toMatchingLogic(def);
+                    // If we are skipping gaps, skip between productions
+                    const named = new NamedMatcher(stepName,
+                        this.$skipGaps === true ? new Break(m, true) : m);
```

Comment:
is the effect here to include the skipped part in the match? that's not what I would have guessed from "skipGaps"